### ISSUE TITLE
This repo is broken on case-insensitive systems

### ISSUE THREAD

**Issue by andreasabel** [haskell/stm#84]
```
$ git clone git@github.com:haskell/stm.git 
Cloning into 'stm'...
...
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'bench/ChanBench.hs'
  'bench/chanbench.hs'
```
Any Linux-capable maintainer, please fix this to include contributors from other cultural contexts.
ATTN: @bgamari

**Comment by hsyl20** [comment]
Fix in #85

**Comment by brandonchinn178** [comment]
Thanks for fixing this! I ticketed this in Gitlab: https://gitlab.haskell.org/ghc/packages/stm/-/issues/2

I wasn't aware this was primarily a GitHub repo. Maybe the gitlab repo should say that issues should be raised in GitHub?
